docs/src/main/sphinx/release/release-409.md

```diff
@@ -0,0 +1,86 @@
+  property is now deprecated in favor of it. ({issue}`15927`)
+* Remove the `distributed-index-joins-enabled` configuration property and
+  related functionality. ({issue}`15375`)
+* Fix failure when using non-comparable and non-sortable values as part of a
+  `VALUES` expression. ({issue}`16242`)
+
+## BigQuery connector
+
+* Add support for inserting default values. ({issue}`16327`)
+* Fix failure when non-lowercase column names exist in the `query` table
+  function. ({issue}`16075`)
+
+## ClickHouse connector
+
+* Remove support for the `ru.yandex.clickhouse.ClickHouseDriver` legacy JDBC
+  driver in the `clickhouse.legacy-driver` configuration property. ({issue}`16188`)
+* Remove support for specifying expressions in the `sample_by` table property to
+  prevent SQL injection. ({issue}`16261`)
+
+## Delta Lake connector
+
+* Automatically infer required Delta Lake version when creating new tables or
+  enabling new table features. ({issue}`16310`)
+* Fix query failure when reading Parquet files generated by Kafka Connect. ({issue}`16264`)
+
+## Hive connector
+
+* Add support for the Hadoop `DefaultCodec` to Hive formats. ({issue}`16250`)
+* Add a native CSV file format reader and writer. These can be disabled with the
+  `csv_native_reader_enabled` and `csv_native_writer_enabled` session properties
+  or the `csv.native-reader.enabled` and `csv.native-writer.enabled`
+  configuration properties. ({issue}`15918`)
+* Add a native JSON file format reader and writer. These can be disabled with
+  the `json_native_reader_enabled` and `json_native_writer_enabled` session
+  properties or the `json.native-reader.enabled` and
+  `json.native-writer.enabled` configuration properties. ({issue}`15918`)
+* Add a native text file format reader and writer. These can be disabled with
+  the `text_file_native_reader_enabled` and `text_file_native_writer_enabled`
+  session properties or the `text-file.native-reader.enabled` and
+  `text-file.native-writer.enabled` configuration properties. ({issue}`15918`)
+* Add a native sequence file format reader and writer. These can be disabled
+  with the `sequence_file_native_reader_enabled` and
+  `sequence_file_native_writer_enabled` session properties or the
+  `sequence-file.native-reader.enabled` and
+  `sequence-file.native-writer.enabled` configuration properties. ({issue}`15918`)
+* Add a native regex file format reader. The reader can be disabled with the
+  `regex_native_reader_enabled` session property or the
+  `regex.native-reader.enabled` configuration property. ({issue}`15918`)
+* Add `regex` and `regex_case_insensitive` table properties for the `REGEX`
+  format. ({issue}`16271`)
+* Improve performance of queries which read from partitioned Hive tables and
+  write to partitioned tables when table statistics are not generated for the
+  source tables. ({issue}`16229`)
+* Use table statistics generated by Apache Spark when statistics have not been
```

Comment:
"Improve query performance when only table statistics generated by Apache Spark are available. This can be disabled via the xxx config property."